### PR TITLE
Adjust UI based on feedback

### PR DESCRIPTION
**Pull request by conalli**
## Description

Adjusted some elements of UI based on feedback from Andy and general observations.

 - [x] Made buttons on home page bigger, changed login buttons colour to have higher contrast.
 - [x] Made 庭 and about page picture a bit more responsive for smaller screens
 - [x] Reduced white space on MyNiwa page for smaller screens 

## Closes issue(s)

If there any issue that is fixed with this PR, please attach it/them here...

## Screenshots

Screenshot 2021-09-10 at 16 20 08
Screenshot 2021-09-10 at 16 20 28

## Changes include

- [x] Bugfix (non-breaking change that solves an issue)

## Checklist

- [x] I have tested this code

### CODE REVIEW

File: src/pages/Home/index.tsx [hunk 1]
```diff
@@ -43,17 +54,19 @@ export const Home = () => {
...
             {userData.isLoggedIn ? (
               <Button
                 variant="contained"
-                color="primary"
                 onClick={() => linkHandler("/user/myniwa")}
+                size="large"
+                className={classes.loginButton}
               >
-                Get me to my Gardens!
+                Get me to my Garden!
```
Comment (curtisahuang) [inline src/pages/Home/index.tsx:69]: Should we change this to My Niwa?

File: src/pages/Home/index.tsx [hunk 2]
```diff
@@ -6,6 +6,16 @@ import { useUserState } from "../../store/user/useUserState";
...
+import { createStyles, makeStyles, Theme } from "@material-ui/core";
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    loginButton: {
+      margin: "4%",
+      backgroundColor: theme.palette.text.primary,
+      color: theme.palette.background.default
+    }
+  }))
```
Comment (curtisahuang) [inline src/pages/Home/index.tsx:18]: I like this button much more

**Review by curtisahuang — APPROVED** [review]
Should we change all the "Get me to my gardens" -> Get me to My Niwa"?

**Review by kome12 — APPROVED** [review]
LGTM!

**Review by daiHash — APPROVED** [review]

File: src/pages/Home/index.tsx (hunk 1 above)
Comment (conalli) [reply src/pages/Home/index.tsx:69]: Sure, that sounds good.

**Pull request merged**